tests/test_submit_artifacts.py

```diff
@@ -427,44 +427,45 @@ def test_setup_scan_prereqs(self):
                     submit_artifacts.setup_scan_prereqs(upload_api=upload_api)
                 )
 
-        # Build in progress, abort new build
+        # Build in progress, try to cancel and recreate
         with patch(
             "veracode.submit_artifacts.is_ready_for_new_build", return_value=False
         ):
-            self.assertFalse(submit_artifacts.setup_scan_prereqs(upload_api=upload_api))
+            # Unsuccessful create build
+            with patch("veracode.submit_artifacts.create_build", return_value=True):
+                # cancel_build returns True, create_build returns False, return False
```

Comment:
```suggestion
                # setup_scan_prereqs returns True when create_build and cancel_build return True
```

Reply:
Looks like I missed these comment suggestions, just fixed for clarity.